lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -40,24 +40,42 @@ class LighthouseViewerReport {
     this.fileUploader = new FileUploader(this.onFileUpload);
     this.github = new GithubAPI();
 
-    this.addListeners();
+    this._isNewReport = true;
+
+    this.initUI();
     this.loadFromURL();
   }
 
-  addListeners() {
+  initUI() {
     const printButton = document.querySelector('.js-print');
     if (printButton) {
       printButton.addEventListener('click', _ => {
         window.print();
       });
     }
 
-    const shareButton = document.querySelector('.js-share');
-    if (shareButton) {
-      shareButton.addEventListener('click', this.onShare);
+    this.shareButton = document.querySelector('.js-share');
+    if (this.shareButton) {
+      this.shareButton.addEventListener('click', this.onShare);
```

Comment:
`pointer-events: none;` to disable click events does seem a little weird way, honestly. Since it's a real button, what about setting `disabled` on it to true?